flocx_market/templates/error.html

```diff
@@ -0,0 +1,12 @@
+<%inherit file="layout.html" />
```

Comment:
@larsks same for this one then I believe. Should I just remove the entire templates directory?

Reply:
Probably, yeah.  Good catch.